requirements_test.txt

```diff
@@ -1,2 +1,2 @@
-pytest==2.8.3
+pytest==3.0.5
```

Comment:
Just for the record: I just tested this with pytest 2.6.3 and it worked. But we still can keep this change.